### DISCUSSION TITLE
Default Passwords in prefs.config

### DISCUSSION THREAD

**Discussion by jlb6907** [ahrm/sioyek#848] [Ideas]
Hi,

I have a lots of pdf files to read, with the same password.

Is it possible to have a "default_passwords" option in the prefs.config file, to quickly open pdf files protected with the same password ? (like in the [sumatrapdf config file](https://www.sumatrapdfreader.org/settings/settings#DefaultPasswords)).

Thanks a lot.

**Comment by ahrm** [comment]
It is not currently possible.
